Dockerfile

```diff
@@ -1,36 +1,25 @@
-FROM debian:7.8
-MAINTAINER Mike Babineau michael.babineau@gmail.com
+# As of March 18, 2020 - corona virus scare time
 
-ENV \
-    ZK_RELEASE="http://www.apache.org/dist/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz" \
```

Comment:
any particular reason to no longer use these values as environment variables?

Reply:
Upgrading to debian:stretch, also locations of the above files changed, directly consuming values rather than having variables.